kafka_test_setup.sh

```diff
@@ -24,9 +28,24 @@ echo "Starting Kafka broker"
 build/kafka/bin/kafka-server-start.sh -daemon build/kafka/config/server.properties --override advertised.host.name=127.0.0.1 --override log.dirs="${PWD}/build/kafka-logs"
 sleep 10
 
-echo "Downloading Confluent Platform"
-curl -s -o build/confluent_platform.tar.gz http://packages.confluent.io/archive/5.5/confluent-community-5.5.1-2.12.tar.gz
+echo "Setup Confluent Platform"
+if [ ! -e confluent-community-5.5.1-2.12.tar.gz ]; then
```

Comment:
Let's abstract this version of cp to the top of the file, like we do with KAFKA_VERSION

Reply:
Done with commit ba0274779d24fd0d4a1935d7c8fb8dbb75e53e27